### PR TITLE
ICU-21010 make SIPrefix.getIdentifier only for internal use

### PR DESCRIPTION
**Pull request by younies**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21010
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [x] Documentation is changed or added

### CODE REVIEW

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java [hunk 1]
```diff
@@ -304,8 +304,8 @@
         /*
          * Returns the identifier of the prefix.
          *
-         * @draft ICU 68
-         * @provisional This API might change or be removed in a future release.
+         * @internal
+         * @deprecated This API is ICU internal only.
```
Comment (yumaoka) [inline icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:308]: You also need `@Deprecated` annotation to this method, not just API doc `@deprecated`.
For your reference, run `ant checkAPIStatus` to see warnings/errors.

**Review by yumaoka — CHANGES_REQUESTED** [review]
See my comment - add `@Deprecated` annotation please.

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java (hunk 1 above)
Comment (younies) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:308]: Thanks @yumaoka 

I still get this 
```
  [javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder()
  [javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder.apply(com.ibm.icu.impl.number.DecimalQuantity)
  [javadoc] warning: no tag for com.ibm.icu.util.MeasureUnit.SIPrefix.getIdentifier()
  [javadoc] warning: no tag for com.ibm.icu.util.MeasureUnit.SIPrefix.getPower()
```

what is the reason?

Comment (younies) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:308]: done!

but why still a warning for 
```
[javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder()
  [javadoc] warning: no tag for com.ibm.icu.number.Precision.BogusRounder.apply(com.ibm.icu.impl.number.DecimalQuantity)
  ```

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java [hunk 2]
```diff
@@ -567,7 +568,7 @@ public MeasureUnit product(MeasureUnit other) {
      * If this is a SINGLE unit, a list of length 1 will be returned.
      *
      * @return An unmodifiable list of single units
-     * @internal ICU 68 Technology Preview
+     * @draft ICU 68 Technology Preview
```
Comment (younies) [inline icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:571]: @hugovdm @sffc 

This should be `@draft`, right ?

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java (hunk 1 above)
Comment (hugovdm) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:308]: I noticed BogusRounder doesn't need to be public. I'll push a commit to this branch.

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java (hunk 2 above)
Comment (hugovdm) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:571]: Looks like it yes: I find no other provisional & internal methods, I find plenty of provisional & draft methods. Thanks!

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java (hunk 1 above)
Comment (hugovdm) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:308]: My bad: "ant checkAPIStatus" doesn't try to build it... :facepalm: - reverted.

Comment (yumaoka) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:308]: I handled all issues except SIPrefix - https://github.com/unicode-org/icu/pull/1371
I planned to update SIPrefix based on the discussion, but you already went ahead and created PR - so I avoided merge conflict.

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java (hunk 2 above)
Comment (yumaoka) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:571]: @draft should not come with Technology Preview. Please just use `@draft ICU 68`. Thanks.

**Review by yumaoka — CHANGES_REQUESTED** [review]
Please remove `Technology Preview`

File: icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java (hunk 2 above)
Comment (younies) [reply icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java:571]: done

**Comment by yumaoka** [comment]
This is done by #1377. I'm closing this PR without merging.

**Pull request closed without merging**